fpga.chart.py

```diff
@@ -92,28 +92,19 @@ def check():
         return True
 
     def _connect(self, dsn):
-        try:
-            self.conn = psycopg2.connect(dsn)
-            self.conn.autocommit = True
-            with self.conn.cursor() as cursor:
-                cursor.execute('SELECT COUNT(*) FROM swarm64da.get_fpga_stats()')
-                self.fpga_count = cursor.fetchone()[0]
-
-        except Exception:
-            self.conn = None
-            return False
-
-        return True
-
-
-    def _get_connection(self):
         if not self.conn:
-            self._connect(self.dsn)
+            try:
```

Comment:
`try` can be removed for now, if it cannot connect then let the exception happen for now.